examples/mxnet/gcn/README.md

```diff
@@ -25,63 +25,77 @@ Results
 -------
 Run with following (available dataset: "cora", "citeseer", "pubmed")
 ```bash
-DGLBACKEND=mxnet python gcn_spmv.py --dataset cora --gpu 0
+DGLBACKEND=mxnet python3 gcn_spmv.py --dataset cora --gpu 0
 ```
 
 * cora: ~0.810 (paper: 0.815)
```

Comment:
Could we add time measure? We may also want to add the accuracies from non-graph versions - were they included it in the original paper?

Reply:
Yes- it is weird that the original paper mentioned graph convolution as an extension to naive classification, but did not seem to provide results from naive classification. On the other hand, the authors felt very busy beating all the other graph-based SOTAs without discussing how or why their differences were important.

Since this is not part of the original paper, we indeed have no responsibility to establish any new results here.